app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -118,6 +118,15 @@ def single_disability_eligible_for_mas?
     RapidReadyForDecision::Constants::MAS_DISABILITIES.include?(diagnostic_codes.first) && increase_only?
   end
 
+  # return whether all disabilities on this form are rated as not service-connected
+  def disabilities_not_service_connected?
```

Comment:
For extra clarity, consider renaming to `all_disabilities_not_service_connected?`